compiler/main/driver.cpp

```diff
@@ -378,6 +378,15 @@ static void setChapelDebug(const ArgumentDescription* desc, const char* arg_unus
   printCppLineno = true;
 }
 
+static void verifyStageAndSetStageNum(const ArgumentDescription* desc, const char* arg_unused)
+{
+  int stageNum = stageNumFromStageName(llvmPrintIrStage);
+  if(!stageNum)
```

Comment:
Everywhere else, stageNum is only ever one of the enum values. So shouldn't this say
``` c
if (stageNum == llvmStageNum::NOPRINT)
```

?